code/style.css

```diff
@@ -1,4 +1,437 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+*{
+  box-sizing: border-box;
 }
+
+figure {
+  margin: 0;
+}
+
+.pagewraper{
+  max-width: 1300px;
+  margin-left: auto;
+  margin-right: auto;
+  width:100%;
+}
+
+.header {
+  background-color:white;
+  border-bottom: 2px solid grey;
+}
+
+.header__title{
+  font-size: 60px;
+  font-weight: bolder;
+  color: black;
+  margin: 0;
+  padding: 10px;
+  font-family:'Cormorant Garamond', serif;
+  text-decoration: none;
+  white-space:nowrap;
+  flex:1;
+}
+
+@media screen and (max-width: 900px){
+  .header__title{
+    width:100%;
+    text-align: center;
+  }
+}
+
+
+@media screen and (max-width:415px){
+  .header__title{
+    white-space: normal;
+    font-size: 51px;
+  }
+}
+
+
+.header__title:visited{
+text-decoration: black;
+}
+
+.header__title:hover{
+  color: blue;
+}
+
+.header__top{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+@media screen and (max-width: 900px){
+  .header__top{
+    flex-wrap: wrap;
+  }
+}
+
+@media screen and (max-width: 900px){
+  .header__title{
+    flex:auto;
+  }
+}
+
+
+.header__social {
+  flex:1;
+}
+
+@media screen and (max-width: 900px){
+  .header__social{
+    order: -1
+  }
+}
+
+.header__actions__item{
+  animation-name: itemmove;
+  animation-duration: 5s;
+  animation-delay: 1s;
+}
+@keyframes itemmove{
+    0%   {background-color: grey;}
+    25%  {background-color: purple;}
+    50%  {background-color: blue;}
+    100% {background-color: red;}
+
+    0%   {color:black;}
+    25%  {color: white;}
+    50%  {color:white;}
+    100% {color: white;}
+}
```

Comment:
This is a very nice use of animation, good job here. If you wanna continue to explore the world of animations in code, check out https://animista.net/ and play around!